.github/workflows/check-build.yml

```diff
@@ -17,16 +17,21 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
         exclude:
```

Comment:
Let's drop 3.6? It's [EOLed](https://endoflife.date/python).

Reply:
I would say

- Let's merge this PR as is.
- Let's merge the other PRs that are waiting just for the CI to work.
- Let's deploy sacrebleu at PyPI, so that anyone who needs 3.6 can still benefit from it.
- We can drop 3.6 immediately after this is done (which will simplify this check-build.yml).